### ISSUE TITLE
yarn create-nuxt-app doesn't install babel-core dependency ?

### ISSUE THREAD

**Issue by lambda0xff** [nuxt/create-nuxt-app#305]
Creating a brand new project with `yarn/npx create-nuxt-app` doesn't install the `babel-core` dependency ?

**Comment by magpiedigital** [comment]
I'm also seeing this.

When running `npm run test`, I got an initial error saying "Cannot find module 'babel-core'"

Fixed by doing `npm i -D babel-core` and the test succeeds, but I still get a coverage error:
```
Running coverage on untested files...Failed to collect coverage from /Users/hamishjohnson/Desktop/test-conf
ig-2/pages/index.vue
ERROR: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel. (While processing preset: "/Users/hamishjohnson/Desktop/test-config-2/node_modules/@babel/preset-env/lib/index.js")
STACK: Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel. (While processing preset: "/Users/hamishjohnson/Desktop/test-config-2/node_modules/@babel/preset-env/lib/index.js")
    at throwVersionError (/Users/hamishjohnson/Desktop/test-config-2/node_modules/@babel/helper-plugin-utils/lib/index.js:65:11)
    at Object.assertVersion (/Users/hamishjohnson/Desktop/test-config-2/node_modules/@babel/helper-plugin-utils/lib/index.js:13:11)
    at _default (/Users/hamishjohnson/Desktop/test-config-2/node_modules/@babel/preset-env/lib/index.js:95:7)
    at /Users/hamishjohnson/Desktop/test-config-2/node_modules/@babel/helper-plugin-utils/lib/index.js:19:12
    at /Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/file/options/option-manager.js:317:46
    at Array.map (<anonymous>)
    at OptionManager.resolvePresets (/Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
    at OptionManager.mergePresets (/Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
    at OptionManager.mergeOptions (/Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
    at OptionManager.init (/Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/Users/hamishjohnson/Desktop/test-config-2/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at compileBabel (/Users/hamishjohnson/Desktop/test-config-2/node_modules/vue-jest/lib/compilers/babel-compiler.js:23:21)
    at processScript (/Users/hamishjohnson/Desktop/test-config-2/node_modules/vue-jest/lib/process.js:30:10)
    at Object.module.exports [as process] (/Users/hamishjohnson/Desktop/test-config-2/node_modules/vue-jest/lib/process.js:42:18)
    at ScriptTransformer.transformSource (/Users/hamishjohnson/Desktop/test-config-2/node_modules/@jest/transform/build/ScriptTransformer.js:387:35)
    at _default (/Users/hamishjohnson/Desktop/test-config-2/node_modules/@jest/reporters/build/generateEmptyCoverage.js:57:9)
    at Object.worker (/Users/hamishjohnson/Desktop/test-config-2/node_modules/@jest/reporters/build/coverage_worker.js:80:45)
    at execFunction (/Users/hamishjohnson/Desktop/test-config-2/node_modules/jest-worker/build/workers/processChild.js:155:17)
    at execHelper (/Users/hamishjohnson/Desktop/test-config-2/node_modules/jest-worker/build/workers/processChild.js:139:5)
    at execMethod (/Users/hamishjohnson/Desktop/test-config-2/node_modules/jest-worker/build/workers/processChild.js:143:5)
    at process.on.request (/Users/hamishjohnson/Desktop/test-config-2/node_modules/jest-worker/build/workers/processChild.js:64:7)
    at process.emit (events.js:198:13)
    at emit (internal/child_process.js:832:12)

```

**Comment by felubra** [comment]
Same here.

**Comment by magpiedigital** [comment]
managed to fix both issues by reinstalling as babel-bridge `npm install babel-core@7.0.0-bridge.0`

**Comment by clarkdo** [comment]
Latest nuxt is using @bable/core, could you please show me your create-nuxt-app version and all selected options so that I can reproduce it?

**Comment by magpiedigital** [comment]
Selecting options
<img width="373" alt="Screenshot 2019-08-04 at 08 32 12" src="https://user-images.githubusercontent.com/9589803/62418424-ae769c00-b692-11e9-8ac7-6fee276dd7b1.png">

Test error
<img width="568" alt="Screenshot 2019-08-04 at 08 32 36" src="https://user-images.githubusercontent.com/9589803/62418438-eed61a00-b692-11e9-90d0-51bc2adc79b8.png">

Package.json
<img width="373" alt="Screenshot 2019-08-04 at 08 33 09" src="https://user-images.githubusercontent.com/9589803/62418444-ff869000-b692-11e9-86c9-a6f0bf9a5541.png">

**Comment by clarkdo** [comment]
Fix has been released as 2.9.1
